src/components/ui/badge.tsx

```
@@ -24,13 +25,38 @@ const badgeVariants = cva(
-    <div className={cn(badgeVariants({ variant }), className)} {...props} />
+    <div className={cn(badgeVariants({ variant }), className)} {...props}>
+      {children}
+      {dismissable && onClose && (
+        <button
+          onClick={(closeEvent) => {
+            closeEvent.stopPropagation();
+            onClose();
+          }}
+          disabled={onClosePending}
+          className="ml-2 rounded-full outline-none ring-offset-background focus:ring-2 focus:ring-ring focus:ring-offset-2"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)